app/views/users/show.html.haml

```diff
@@ -0,0 +1,28 @@
+%h1 Users
+
+%table.table-striped.table-bordered
+  %tr
+    %th Fullname
+    %th Actions
+  - @users.each do |user|
+    %tr
+      %td= user.fullname
+      %td
+        = link_to 'Trim', trim_user_path(user), :id => "#{user.id}", :class => 'trim', :data => {:url => '/trim'}
```

Comment:
Hi Bram,

Does this still need the method: :put parameter since I'm using ajax and preventing the default anyway?